all_test.go

```diff
@@ -325,6 +324,20 @@ func isPrefix(prefix, str string) bool {
 	return strings.Index(str, prefix) == 0
 }
 
+type TestNonLockableUser struct {
+}
+
+func (x TestNonLockableUser) IsLockable(identity string) (bool, error) {
+	return true, nil
```

Comment:
Why does IsLockable return true, when the user is NOT lockable?
A function's return value must be true to its name.